ci/Jenkinsfile

```diff
@@ -62,16 +62,18 @@ pipeline {
 
   environment {
     QT_QPA_PLATFORMTHEME = 'qt5ct'
-    QT_LOGGING_DEBUG = 1
-    QT_DEBUG_PLUGINS = 1
 
     SQUISH_DIR = '/opt/squish-runner-7.1-20230222-1555'
     PYTHONPATH = "${SQUISH_DIR}/lib:${SQUISH_DIR}/lib/python:${PYTHONPATH}"
     LD_LIBRARY_PATH = "${SQUISH_DIR}/lib:${SQUISH_DIR}/python3/lib:${LD_LIBRARY_PATH}"
 
+    /* Avoid weird bugs caused by stale cache. */
+    QML_DISABLE_DISK_CACHE = "true"
```

Comment:
This is a build-time flag, so we don't need it here:

>Disables the disk cache and forces re-compilation from source for all QML and JavaScript files. QML_DISABLE_DISK_CACHE overrides QML_DISK_CACHE.

https://doc.qt.io/qt-6/qmldiskcache.html